packages/sample-app/sample_app/multiple_span_proccesors.py

```
@@ -0,0 +1,36 @@
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+from traceloop.sdk import Traceloop
+from traceloop.sdk.tracing.context_manager import get_tracer
```

Ruff (0.12.2)

`traceloop.sdk.tracing.context_manager.get_tracer` imported but unused

Remove unused import: `traceloop.sdk.tracing.context_manager.get_tracer`

(F401)